rules/windows/builtin/windefend/win_defender_restored_quarantine_file.yml

```diff
@@ -0,0 +1,21 @@
+title: Win Defender Restored Quarantine File
+id: bc92ca75-cd42-4d61-9a37-9d5aa259c88b
+status: experimental
+description: Detects the restoration of files from the defender quarantine
+references:
+    - https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/troubleshoot-microsoft-defender-antivirus?view=o365-worldwide
+author: Nasreddine Bencherchali
+date: 2022/12/06
+tags:
+    - attack.defense_evasion
+    - attack.t1562.001
+logsource:
+    product: windows
+    service: windefend
+detection:
+    selection:
+        EventID: 1009 # The antimalware platform restored an item from quarantine.
+    condition: selection
+falsepositives:
+    - Legitimate administrator activity restoring a file
+level: high
```

Comment:
If defender put in quarantine it means it thinks it's suspicious which is high by default since we have (57b649ef-ff42-4fb0-8bf6-62da243a1708) this rule that will trigger when defender finds a threat.

Now an admin/attacker restoring something from quarantine is a decision that needs to be evaluated. Why do you think its an FP? are you testing something malicious or is it truly and FP? 

Hence it's high imo.

Reply:
I see your point. I am thinking about the case where it is indeed a FP, you got alerted and evaluated the finding. The verdict is: It is a FP and you make the file available for the user again. Now you get another alert for a case you already solved.

On the other hand I agree that in the true positive case the criticality is even higher than just the windefend match.

@Neo23x0 : Can you weigh in? :innocent:

Reply:
As I see it, it is a typical event that should be filtered based on the preferences of the organisation. 
In some organisations it can highly relevant, in others it isn't. We keep it on level high and let the users decide.